canary/Models/Record.cs

```diff
@@ -199,10 +199,19 @@ public static (Record record, List<Dictionary<string, string>> issues) CheckGet(
 
             if (e.Message != null)
             {
-                foreach (string er in e.Message.Split(";"))
+                Exception baseException = e.GetBaseException();
+
+                foreach (string er in baseException.Message.Split(";"))
```

Comment:
If my previous comment isn't addressed then could baseException.Message ever be null?

Reply:
I do not believe so however for clarity I have gone ahead and migrated all the code over to use baseException in my last commit.